static/app/utils/useTeams.tsx

```diff
@@ -125,6 +134,53 @@ function useTeams({limit}: Options = {}) {
     fetchError: null,
   });
 
+  async function loadUserTeams() {
+    const orgId = organization?.slug;
+    if (orgId === undefined) {
+      return;
+    }
+
+    setState({...state, fetching: true});
+    try {
+      await fetchUserTeams(api, {orgId});
```

Comment:
What happens if they have more than 100 teams?

Reply:
This user-teams endpoint doesn't paginate, so we would fetch every single team.

Reply:
This would only happen once though, because after that the store would be loaded with those teams.

Reply:
Ah yeah cool